src/sql/workbench/contrib/notebook/browser/cellViews/code.component.ts

```diff
@@ -355,7 +355,7 @@ export class CodeComponent extends CellView implements OnInit, OnChanges {
 				horizontalScrollbar.style.top = horizontalTop + 'px';
 				horizontalScrollbar.style.bottom = '';
 			}
-		} else {
+		} else if (this.cellModel.cellType !== CellTypes.Code) {
 			// If horizontal scrollbar is not needed then set do not show it
 			horizontalScrollbar.style.opacity = '0';
```

Comment:
Why are we even using opacity to hide the scrollbar? I would think `display: none` would be the better option

@VasuBhog

Reply:
Let's make sure we aren't regressing the scenarios this was originally added for too : https://github.com/microsoft/azuredatastudio/pull/17083

Reply:
I believe that `display:none` could be used here so that the dom layout would will not need to compute the layout for the horizontal scrollbar. I believe when I was testing all the paths it was much easier to have this change so that I was able to still see the scrollbar in the DOM in order to change it if necessary. 

@unlock21 please change to using display instead :)

Reply:
@VasuBhog changed it to use display.

You all got to this way faster than I'd expected, thank you!